### PR TITLE
Fix golint failure in pkg/kubectl/cmd/certificates

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
This fixes golint failures of the following file:

- test/utils/conditions.go
- test/utils/runners.go
- pkg/kubectl/cmd/certificates/

**Which issue(s) this PR fixes**:
Ref [#68026](https://github.com/kubernetes/kubernetes/issues/68026)

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
None
```

### CODE REVIEW

**Comment by ZP-AlwaysWin** [comment]
/assign @janetkuo
hi janekuo
Help me, please review my code

Best wishes

**Comment by ZP-AlwaysWin** [comment]
/ test

**Comment by ZP-AlwaysWin** [comment]
/assign @soltysh

hi @soltysh

Best wishes

File: pkg/kubectl/cmd/certificates/certificates.go [hunk 1]
```diff
@@ -68,15 +68,15 @@ type CertificateOptions struct {
 	genericclioptions.IOStreams
 }
 
-// NewCertificateOptions creates the options for certificate
-func NewCertificateOptions(ioStreams genericclioptions.IOStreams) *CertificateOptions {
-	return &CertificateOptions{
+// newcertificateOptions creates the options for certificate
```
Comment (draveness) [inline pkg/kubectl/cmd/certificates/certificates.go]: s/newcert/newCert

File: pkg/kubectl/cmd/certificates/certificates.go [hunk 2]
```diff
@@ -103,15 +103,15 @@ func (o *CertificateOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, arg
...
 	if len(o.csrNames) < 1 && cmdutil.IsFilenameSliceEmpty(o.Filenames, o.Kustomize) {
 		return fmt.Errorf("one or more CSRs must be specified as <name> or -f <filename>")
 	}
 	return nil
 }
 
-func NewCmdCertificateApprove(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
-	o := NewCertificateOptions(ioStreams)
+func newCmdCertificateApprove(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	o := newcertificateOptions(ioStreams)
```
Comment (draveness) [inline pkg/kubectl/cmd/certificates/certificates.go]: same here

File: test/utils/conditions.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ import (
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 )
 
-type ContainerFailures struct {
+type containerFailures struct {
```
Comment (draveness) [inline test/utils/conditions.go]: Could you explain the reason for un-export this struct?

Comment (ZP-AlwaysWin) [reply test/utils/conditions.go]: Hi @draveness

Thank for review my first pull request for k8s.
ContainerFailures struct called only in this file test/util/conditions.go

“s/newcert/newCert”

The above modification is also based on this reason.

Best wishes

Comment (draveness) [reply test/utils/conditions.go]: > ContainerFailures struct called only in this file test/util/conditions.go

SGTM

> “s/newcert/newCert”
> 
> The above modification is also based on this reason.

Please make the "c" to uppercase, which is not the first letter in the function.

**Review by draveness — COMMENTED** [review]
/release-note-none
/priority backlog

We don't use merge in kubernetes development, and please squash the commits into one,

**Review by ZP-AlwaysWin — COMMENTED** [review]
/test

**Comment by ZP-AlwaysWin** [comment]
/assign @draveness

Hi @draveness

Because the #79782 commit was submitted many times and there were too many conflicts, I will close it and resubmitted it.

#79829

Thank for review my first pull request for k8s。
Thank you very much

Best wishes

**Pull request closed without merging**